### PR TITLE
Make fragment identifier links visible below the header.

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

**Comment by Hixie** [comment]
Fixes https://github.com/flutter/flutter/issues/10525

File: _sass/_layout.scss [hunk 1]
```diff
@@ -191,3 +191,9 @@
         }
     }
 }
+
+/* Make fragment identifier links visible below the header. */
+:target:before {
+    content: "";
+    display: block;
+    height: 112px; }
```
Comment (yjbanov) [inline _sass/_layout.scss]: nit: the rest of the file places closing `}` on the next line.

**Comment by yjbanov** [comment]
In fact, RSLGTM!

File: _sass/_layout.scss (hunk 1 above)
Comment (Hixie) [reply _sass/_layout.scss]: Well that's messed up. That's how I had it, but then I noticed the rest of the file it was in was different, so I changed it, then found it was the wrong file, then moved it to this file, and the style is different. -_-

**Comment by sethladd** [comment]
thank you!

**Pull request merged**